app/controllers/users_controller.rb

```diff
@@ -93,6 +93,16 @@ def impersonate
     end
   end
 
+  def invalidate_jwt
+    @user = find_resource(:edit_users)
+    User.find_by(id: @user.id).jwt_secret&.destroy
+    if @user.jwt_secret.blank?
+      process_success(
+        :success_msg => _('Successfully invalidated JWT for %s.') % @user.login
```

Comment:
```suggestion
        :success_msg => _('Successfully invalidated JWTs for %s.') % @user.login
```
We're invalidating _all_ of user's JWTs, right?

Reply:
Each user can have only one JWT. In my opinion, we should keep it singular.